CMakeLists.txt

```diff
@@ -38,6 +38,7 @@ option(GEARSHIFFT_HCFFT "< Not implemented yet >" OFF)
 set(GEARSHIFFT_NUMBER_WARM_RUNS "10" CACHE STRING "Number of repetitions of an FFT benchmark after a warmup.")
 set(GEARSHIFFT_NUMBER_WARMUPS "2" CACHE STRING "Number of warmups of an FFT benchmark.")
 set(GEARSHIFFT_ERROR_BOUND "0.00001" CACHE STRING "Error-bound for FFT benchmarks.")
+set(GEARSHIFFT_INSTALL_CONFIG_PATH "/usr/share/gearshifft/" CACHE STRING "Default install path of config files.")
```

Comment:
@tdd11235813 a better default would be `... "${CMAKE_INSTALL_PREFIX}/share/gearshifft/" CACHE ...`